src/models/bonw15/internal-overview.md

```diff
@@ -0,0 +1,27 @@
+# Bonobo WS (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- LCD panel connector is highlighted in light green
+- Webcam/microphone connector is highlighted in cyan
+- Keyboard adhesive access point is highlighted in pink
+- Wireless card (under the innermost SSD) is highlighted in navy
+- M.2 SSDs are highlighted in dark green
+    - Both slots support PCIe NVMe Gen 4
+- Fan connectors are highlighted in orange
+- CMOS battery connector is highlighted in purple
+- Speaker connectors are highlighted in yellow
+    - Right speaker connector (on the left) is under SSD slot 3
+- BIOS flash chip (U58) is highlighted in black
+- Main battery connector is highlighted in red
+- Touchpad connector is highlighted in white
+- RAM is highlighted in maroon
+
+## Under-keyboard components and connectors:
+
+![Under-Keyboard Connectors](./img/under-keyboard.webp)
+
+- Keyboard connector is highlighted in yellow
+- Keyboard backlight connectors are highlighted in green
```

Comment:
Probably best to also mark the power button connector and the adhesive access point, like in addw3 and gaze18.

Reply:
Fixed with https://github.com/system76/tech-docs/pull/176/commits/e2544d2cbcd090bbbcdbb6a1b396b9d7facd85d3.